pkg/fifo/fifo.go

```diff
@@ -1,101 +1,97 @@
 package fifo
 
 import (
-	"github.com/Dirk007/simpleFifo/pkg/fifo/internal/item"
+	"github.com/Dirk007/simpleFifo/pkg/fifo/implementations"
```

Comment:
a fifo gets called like `fifo.NewFifo().WithImplementation(implementations.NewDoubleLinkedFiFo())` not shure if this isn't a bit wordy

Reply:
bessere Idee? Ich bin da leidenschaftslos.